IDGPredict/FacetImage.h

```diff
@@ -107,7 +116,7 @@ class FacetImage
 		return clippedFacet;
 	}
 	
-	ao::uvector<double> _data;
+	std::vector<ao::uvector<double>> _data;
```

Comment:
Maybe comment here what the vectors do (e.g. For each spectral term, uvector with ...)

Reply:
Good idea